susemanager-utils/susemanager-sls/src/modules/mgractionchains.py

```diff
@@ -178,27 +209,47 @@ def resume():
 
     This method is called by the Salt Reactor as a response to the 'minion/start/event'.
     '''
-    next_chunk = _read_next_ac_chunk()
-    if not next_chunk:
+    yaml_file = _read_next_ac_chunk()
+    if not yaml_file:
         return {}
-    if type(next_chunk) != dict:
+    if type(yaml_file) != dict:
```

Comment:
I think `yaml_file` is a bit misleading as a name when it has to be a `dict`. I would expect a file-like object is bound to a variable that's name like that.